plugins/neo4j-graph-widget/core/CoreExtension.js

```diff
@@ -51,10 +51,14 @@ export class CoreExtension {
     return response
   }
 
-
-  async request(url, config = {}) {
+  getFullUrl(url) {
     const Api = this.#core.api.constructor
```

Comment:
I think you can return directly the full URL 
```suggestion
    return this.#core.api.constructor.getFullUrl(url)
```